spacy/pipeline/transition_parser.pyx

```diff
@@ -30,6 +30,9 @@ from ..errors import Errors, Warnings
 from .. import util
 
 
+_NUMPY_OPS = NumpyOps()
```

Comment:
I suggest to call this `NUMPY_OPS` for consistency across Thinc & spaCy? We've never used the underscore before...

Reply:
I'll change this. I though to underscore it to prevent this to be considered public.

Reply:
Fixed.